S/seqgen/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "SeqGen"
+version = v"1.3.4"
+
+# Collection of sources required to build SFML
+sources = [
+    GitSource("https://github.com/rambaut/Seq-Gen.git", "bc9d8070b2cd1f1352f74282e5b209302eae38a1")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/Seq-Gen/source
+head -39 seq-gen.c > license
+install_license ${WORKSPACE}/srcdir/Seq-Gen/source/license
+
+# compile seq-gen
+make -j${nproc}
+
+# move to bindir
+mkdir ${bindir}
+
+if [[ "${target}" == *-mingw32* ]]; then
+    # not entirely sure this will work on windows
+    # but it is wourth a shot
+    cp seq-gen ${bindir}/seq-gen.exe
+else
+    cp seq-gen ${bindir}/seq-gen
+fi
```

Comment:
You can simplify this a lot by using the [automatic environment variables](https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables)
```suggestion
install -Dvm 755 "seq-gen${exeext}" "${bindir}/seq-gen${exeext}"
```